Automattic/jetpack
# CRM: Learn More buttons and links to be consistent with Jetpack style guide

## Proposed changes:

* This PR changes the style of the Learn More button to be an info icon in line with Jetpack styles and following the Jetpack CRM style guide.
* It also changes the Learn More link in the Learn More popup to include an external link icon.

### Other information:


## Jetpack product discussion

https://github.com/Automattic/zero-bs-crm/issues/2998

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

* Using the [Jetpack Beta plugin](https://github.com/Automattic/jetpack-beta) on a test site with this branch applied or testing locally with the branch checked out, open up any page that previously had a Learn More button in the navigation (for example the Dashboard, any Contact / Transaction / Company / Invoice / Quote dashboard pages).
* You should now see an info icon in place of the Learn More button.
* On clicking the icon, the popup should include a Learn More link followed by an external link icon.

Info icon style: 

<img width="289" alt="Screenshot 2023-04-17 at 16 50 26" src="https://user-images.githubusercontent.com/16754605/232541248-7ae235f0-8d3c-42ef-8ebf-7eb197acf730.png">


<img width="976" alt="Screenshot 2023-04-17 at 16 50 35" src="https://user-images.githubusercontent.com/16754605/232541195-50ab3bc9-a595-4013-b9c5-cfcb54a9adf3.png">
